src/utils/utils.ts

```diff
@@ -150,3 +150,12 @@ export const sanitizeStringForAttribute = (str: string) => {
   const sanitizedStr = str.replace(/[^a-z0-9]/gi, "-").toLowerCase();
   return sanitizedStr;
 };
+
+export const arrayToCommaList = (arr: []) => {
+  let newArray = [];
+  arr.forEach((val: string) => {
+    let item = `\`"${val}"\``;
+    newArray.push(item);
+  });
+  return newArray.join(", ");
+};
```

Comment:
@EdwinGuzman 
I added this untility function to help generate lists from existing arrays—rather than manually updating the lists.  The function does what I want, but the backticks are not rendering the code blocks in the MDX file.  It woudl make sense that this problem is due to the run-time genration of the elements.  Any thoughts on how to get the backticks to render as expected?

Reply:
I tried but couldn't get it to work. You're right, it's mostly a  run-time generation issue that we cannot easily get around.